R/CohortLevelDiagnostics.R

```diff
@@ -57,6 +57,15 @@ getCohortCounts <- function(connectionDetails = NULL,
     counts <-
       DatabaseConnector::querySql(connection, sql, snakeCaseToCamelCase = TRUE) %>%
       tidyr::tibble()
+
+    if (length(cohortIds) > 0) {
+      cohortIdDf <- tidyr::tibble(cohortId = cohortIds)
+      counts <- cohortIdDf %>%
+        dplyr::left_join(counts, by = "cohortId") %>%
+        dplyr::mutate(cohortEntries = ifelse(is.na(cohortEntries), 0, cohortEntries),
```

Comment:
this is probably my personal choice, this tidyr function is more readable for me

https://tidyr.tidyverse.org/reference/replace_na.html